radix-engine-interface/src/api/types/re_node.rs

```diff
@@ -194,6 +204,13 @@ pub enum WorktopOffset {
     Worktop,
 }
 
+#[derive(Debug, Clone, TypeId, Encode, Decode, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub enum ClockOffset {
+    CurrentTimeInMillis,
+    CurrentTimeInSeconds,
+    CurrentTimeInMinutes,
```

Comment:
I think we had decided in an earlier builders meeting to just do `Minutes` as I think having too many variations can make things confusing

Reply:
Hmh, wasn't that about what we expose in scrypto, while still keeping more granular substates around (possibly for future use)? So far in this PR I've only exposed the time rounded to minutes

Reply:
I don't think that was the result of our convo back then. I'd definitely much rather have this simplified and just have a single substate for time, atleast as the first pass.